tardis-setups/setup_utils/download.py

```diff
@@ -0,0 +1,13 @@
+import requests
```

Comment:
TARDIS has a function to download from URL, maybe works here!

https://github.com/tardis-sn/tardis/blob/master/tardis/io/util.py#L390